.env.example

```
@@ -1,2 +1,8 @@
 # Go to -> https://docs.trustlesswork.com/trustless-work/developer-resources/request-api-key
 NEXT_PUBLIC_API_KEY=""
+
+# ROZO Integration (server-side only — NEVER expose in frontend)
+# Get credentials from Trustless Work team — see issue #26
+ROZO_API_KEY=
+ROZO_APP_ID=rozoTrustlesswork
+ROZO_API_BASE_URL=https://intentapiv4.rozo.ai/functions/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ROZO_API_BASE_URL key should go before the ROZO_API_KEY key

(UnorderedKey)